.env.example

```diff
@@ -48,9 +51,11 @@ ETHERSCAN_API_KEY=
 # These need to be deployed and set manually, ideally from circle's stablecoin-evm repo
 
 # Mainnets
+MAINET_USDC_IMPLEMENTATION=
```

Comment:
unused env variable. Why do we need it here?

Reply:
https://github.com/defi-wonderland/opUSDC/pull/184/files#diff-270cf5acd09109c3f031cedfaa912f9ff5e1fb1894f41800518d70bd6519ffaaR11

I think this is the same variable.